templates/web/base/admin/reports/_edit_main.html

```diff
@@ -30,6 +30,16 @@
 [% IF problem.used_map %][% loc('used map') %][% ELSE %][% loc("didn't use map") %][% END %])</small>
 </li>
 
+[% # Peterborough staff only %]
```

Comment:
Because this (and the ones in `information.html` and `_item_small.html`) only apply to Peterborough it would be better to get them into Peterborough-specific templates somehow instead of doing a cobrand check in the base templates.

Generally if the template is fairly small then it's OK to override the entire thing in the cobrand dir - I'd say in this case that applies to `_item_small.html`.

For bigger templates like this one and `information.html` the `[% TRY %][% INCLUDE 'foo.html' %][% CATCH file %][% END %]` pattern allows cobrands to provide a smaller included template that provides custom markup.

Reply:
I've ended up adding a single template shared across all three pages; it seemed the most succinct way.